core/src/main/java/com/netflix/iceberg/avro/AvroSchemaUtil.java

```diff
@@ -87,7 +98,7 @@ public static Schema buildAvroProjection(Schema schema, com.netflix.iceberg.Sche
 
   public static boolean isTimestamptz(Schema schema) {
     LogicalType logicalType = schema.getLogicalType();
-    if (logicalType != null && logicalType instanceof LogicalTypes.TimestampMicros) {
+    if (logicalType instanceof LogicalTypes.TimestampMicros) {
```

Comment:
I'm all for cleaning up minor issues like this, but I would prefer to keep these in separate commits so that the project is easier to maintain. Could you move this sort of improvement into a separate PR that we can get in separately (and probably more quickly)?

Reply:
Sure, will create a separate PR